README.rst

```diff
@@ -789,6 +789,15 @@ Then, add the following command to your CI:
 
     $ vendor/bin/php-cs-fixer fix --config=.php_cs.dist -v --dry-run --using-cache=no --path-mode=intersection `git diff --name-only --diff-filter=ACMRTUXB $COMMIT_RANGE`
 
+If filenames contain spaces:
+
+.. code-block:: bash
+
+    $ IFS=$'\n'
+    $ files=($(git diff --name-only --diff-filter=ACMRTUXB $COMMIT_RANGE))
+    $ unset IFS
+    $ vendor/bin/php-cs-fixer fix --config=.php_cs.dist -v --dry-run --using-cache=no --path-mode=intersection "${files[@]}"
```

Comment:
yes, pls refer to https://github.com/koalaman/shellcheck/wiki/SC2086 and https://github.com/koalaman/shellcheck/wiki/SC2089